pkg/fedramp/config.go

```diff
@@ -83,11 +88,12 @@ var AdminTokenURLs = map[string]string{
 	"integration": fmt.Sprintf("https://rh-ocm-appsre-integration.%s", cognitoURL),
 }
 
-// ClientIDs allows the value of the `--env` option to map to the various AWS Cognito user pool clients.
+// ClientIDs allows the value of the `--env` option to map to the Keycloak clients.
 var ClientIDs = map[string]string{
-	"production":  "72ekjh5laouap6qcfis521jlgi",
-	"staging":     "1lb687dlpsmsfuj53r3je06vpp",
```

Comment:
These still need to be in place, otherwise stage/prod will break. Maybe just a single line change to update integration to `console-dot`.

Reply:
Updated to single line